### DISCUSSION TITLE
Button with CTA - full button link

### DISCUSSION THREAD

**Discussion by betarepos** [arthelokyo/astrowind#259] [Ideas]
With the Astro 3 branch it seems to me, that the buttons that use the CTA are only clickable on the link text from the CTA, could we make the whole button clickable?

**Comment by prototypa** [comment]
Hi @betarepos 

Update the template, I think we have already fixed that. Thanks

**Comment by betarepos** [comment]
you are right, found it in the changes, came right 1 day after i set up my project and i never checked for updates then :D
